python-as-wrapper.yaml

```diff
@@ -0,0 +1,176 @@
+package:
+  name: python-as-wrapper
+  version: 0.1.0
+  epoch: 0
+  description: Shell script wrapper to exec python3.XX
+  copyright:
+    - license: Apache-2.0
+
+# In a "normal" installation of python, 'python3' is a symlink
+# to 'pythonX.Y' where X.Y is Major.Minor (3.13).  In that scenario,
+# if caller invokes 'python3', then when python reads its sys.executable,
+# the executable is 'python3'.
+#
+# That value of sys.executable is "sticky" and finds its way into shbangs that
+# are installed with 'pip install' or other places.
+#
+# In the vast majority of places, we do not want shebang with
+# /usr/bin/python3 because at runtime:
+#  * python3 might be a symlink to another python version
+#  * python3 may not exist at all - with only python3.XX installed.
+#
+# Using the wrapper that this package provides if you do
```

Comment:
I feel this might be a simpler description:

> This wrapper will cause the scripts generated by `python3 -m pip install pkg` to be generated with a shbang of `/usr/bin/python3.Y` instead of `/usr/bin/python3`.

It might also be worth noting (if this is indeed your intent) that:

>  `python3.X -m pip install pkg` should normally be used. Use of these wrappers is intended only as a way to work around packages that still end up generating `/usr/bin/python3?` shbangs.

Reply:
:heavy_check_mark: